test/metabase/models/action_test.clj

```diff
@@ -122,11 +122,17 @@
 (deftest model-to-saved-question-test
   (mt/test-drivers (mt/normal-drivers-with-feature :actions/custom)
     (mt/with-actions-enabled
-      (testing "Actions are archived if their model is converted to a saved question"
-        (mt/with-actions [{:keys [action-id model-id]} {}]
+      (testing "Non-implicit actions are archived if their model is converted to a saved question"
```

Comment:
I tried it and it's only one line shorter, and subjectively harder to read. I don't think it's worth changing

<img width="1012" alt="image" src="https://user-images.githubusercontent.com/39073188/224057463-9ebc2108-51b2-4c95-bfe5-0499b80cf240.png">

Reply:
I actually think you're suggesting something different, like using 
```
(mt/with-actions [<implicit action binding>
                           <query action binding>
                           <http action binding>] 
   ...)
```
but that would end up longer because you have to name each id, and test them independently. It's not worth it either

Reply:
I think it has a nother implication is that it tests case where all 3 actions type exists at once. Tho it's not a big difference.

Reply:
Yeah, I can't see any difference really. I'll leave it